package.json

```diff
@@ -122,6 +120,7 @@
         "serve": "11.3.2",
         "sinon": "10.0.0",
         "terser": "5.7.0",
-        "testem": "3.4.1"
+        "testem": "3.4.1",
+        "xhr-mock": "2.4.1"
```

Comment:
"xhr-mock" is used in the tests, but not copied as a test dependency. Should we just remove all of the copying for test dependencies and source them all from node-modules directly?

Reply:
I've added an npm script for this since, for consistency, we can't copy qunit in since our own version has diverged